BlockEQ/View Controllers/Wallet/AddressEntryViewController.swift

```diff
@@ -70,8 +70,26 @@ final class AddressEntryViewController: UIViewController {
         let availableSendBalance = account.availableSendBalance(for: asset).displayFormattedString
         navigationItem.title = String(format: "TRADE_BALANCE_FORMAT".localized(), availableSendBalance, asset.shortCode)
     }
+    
+    @IBAction func textFieldEditingDidChange(_ sender: Any) {
+        let text = sendAddressTextField.text!
+        if (text.contains("*") && text.contains(".com")) {
```

Comment:
This would be better as an extension on `StellarAddress`. That object has rules that takes an optional string (so you can directly pass the value of a text field into it), and constructs a usable Stellar address from it.

I've been planning to use it to be able to determine if an address is raw, or a federated address with the

```swift
public enum AddressType { ... }
```

object. In that case, this logic can be re-used throughout the app by doing something like:

```swift
public var kind: AddressType {
        /* Your logic to determine if an address is federated or not, returning .normal or .federated */
}
```